lib/signUpScreen/SignUpScreen.tsx

```diff
@@ -0,0 +1,206 @@
+import * as React from "react";
+import {
+  Text,
+  View,
+  Image,
+  TextStyle,
+  SafeAreaView,
+  TouchableOpacity,
+  ViewStyle,
+} from "react-native";
+/**
+ * ? Local Imports
+ */
+import styles from "./SignUpScreen.style";
+import TextField from "../components/TextField/TextField";
+import SocialButton from "../components/SocialButton/SocialButton";
+
+// ? Assets
+const backArrowImage = require("../local-assets/left-arrow.png");
+
+export interface ISocialLoginProps {
+  loginText?: string;
+  loginTextStyle?: TextStyle;
+  backArrowImageSource?: any;
+  rightTopAssetImageSource?: any;
+  signinTitleText?: string;
+  signinTextStyle?: TextStyle;
+  usernameTextFieldStyle?: TextStyle;
+  usernamePlaceholder?: string;
+  onUserNameChangeText?: (text: string) => void;
+  passwordPlaceholder?: string;
+  onPasswordChangeText?: (text: string) => void;
+  passwordTextFieldStyle?: TextStyle;
+  onloginTextPress?: () => void;
+  signinText?: string;
+  signinButtonBackgroundColor?: string;
+  signinButtonShadowColor?: string;
+  signinButtonSpinnerVisibility?: boolean;
+  spinnerSize?: number;
+  spinnerType?: string;
+  signinButtonSpinnerColor?: string;
+  onSigninPress: () => void;
+  leftBottomAssetImageSource?: any;
+  emailPlaceholder?: string;
+  emailTextFieldStyle?: TextStyle;
+  onEmailChangeText?: (text: string) => void;
+  signinButtonContainer?: ViewStyle;
+}
+
+interface IState {}
+
+export default class SignUpScreen extends React.PureComponent<
+  ISocialLoginProps,
+  IState
+> {
+  renderHeader = () => {
+    const {
+      loginText = "LOG IN",
```

Comment:
This is a signup screen, can you change the loginText and all of them to signUp.. Also "LOG IN" should change to "SIGN UP"